pages/Documents/GettingStarted/liveperson-apis-and-sdks.md

```diff
@@ -67,6 +67,16 @@ date_updated: 2022/09/30
 | [Window Customization API](window-customization-api-overview.html) | Customize the look and feel of the web engagement window |
 | [Workdays API](workdays-api-overview.html) | Configure hours of operation for specific days, and assign them to specific skills |
 
+#### Conversational AI APIs
+
+Building a custom automation solution? These Conversational AI APIs might be of interest:
```

Comment:
If I may, how about phrasing this so that it’s a little more affirmative, as with “…can help you”?

As these are Conv AI APIs, I’d assume they do! :)

Reply:
Yes, good idea. Tweaked.